scripts/jenkins/ardana/ansible/roles/input_model_generator/templates/input_model/data/firewall_rules.yml

```diff
@@ -29,39 +29,114 @@
 #
 
   firewall-rules:
+
+{% if ipv6 == true %}
+    - name: SSH_CLM
+      network-groups:
+{% for network_group in scenario['network_groups']
+  if 'CLM' in network_group['component_endpoints'] or
+     'MANAGEMENT' in network_group['component_endpoints'] %}
+         - {{ network_group.name|upper }}
+{% endfor  %}
+      rules:
+      - type: allow
+        remote-ip-prefix:  0.0.0.0/0
```

Comment:
Could you comment on this comment @italiahardik ? Thanks!

Reply:
This rule is only applicable to CLM/MANAGEMENT network and it is still being configured to use ipv4 network for access purpose.

Reply:
Because we will be still using deployment/management network as ipv4 when ipv6 is enabled.
When ipv4 is enabled, it will skip SSH_CLM rule and will only add below SSH rule.